CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [Unreleased]
+### Added
+- Breadcrumbs 
```

Comment:
should we add here information that you've added a global variable `$outline-base`? IMO we should

Reply:
According to the link from alpaca rocket channel (http://keepachangelog.com/en/1.0.0) :
```
The purpose of a commit is to document a step in the evolution of the source code. Some projects clean up commits, some don't.

The purpose of a changelog entry is to document the noteworthy difference, often across multiple commits, to communicate them clearly to end users.
```
I guess we don't have to put this kind of info within this file. I have updated this message, check it out.

Reply:
OK, it's clear inside the commit. lets leave it